GoMain/src/main/main.go

```diff
@@ -80,23 +80,23 @@ func main() {
 		log.Fatalf("[%s] Orchestaration initalize fail : %s", logPrefix, err.Error())
 	}
 
-	externalapi, err := orchestrationapi.GetExternalAPI()
-	if err != nil {
-		log.Fatalf("[%s] Orchestaration external api : %s", logPrefix, err.Error())
-	}
-
-	if err == nil {
-		// @NOTE : for container
-		serviceInfo := make([]orchestrationapi.RequestServiceInfo, 1)
-		serviceInfo[0].ExecutionType = "container"
-		serviceInfo[0].ExeCmd = []string{"docker", "run", "-v", "/var/run:/var/run:rw", "hello-world"}
-		requestService := orchestrationapi.ReqeustService{
-			ServiceName: "container_service",
-			ServiceInfo: serviceInfo,
-		}
-
-		externalapi.RequestService(requestService)
-	}
+	// externalapi, err := orchestrationapi.GetExternalAPI()
```

Comment:
If it is no needed, Please remove it.
and if it was for guidance for use, insert comments

Reply:
Yes, i think it's better to remove it.
Guide of rest-api is in README.